cupy/cuda/device.pyx

```diff
@@ -3,17 +3,13 @@
 import threading
 
 from cupy.cuda import cublas
+from cupy.cuda import cusolver
 from cupy.cuda import cusparse
 from cupy.cuda cimport runtime
 from cupy.cuda import runtime as runtime_module
 from cupy import util
 
-try:
-    from cupy.cuda import cusolver
-    cusolver_enabled = True
-except ImportError:
-    cusolver_enabled = False
-
+cusolver_enabled = True
```

Comment:
Could you add some comment to avoid confusing users? The code looks like it is a configurable flag to disable cuSOLVER.

e.g.,
```
# This flag is kept for backward compatibility.
# It is always True as cuSOLVER library is always available in CUDA 8.0+.
```